backend/schemas/university.py

```
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+from typing import List, Dict
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)